app/models/restaurant.rb

```diff
@@ -0,0 +1,7 @@
+class Restaurant < ApplicationRecord
+  validates :name, presence: true
+  validates :street, presence: true
+  validates :zip_code, presence: true
+  validates :city, presence: true
+  validates :food_style, presence: true
+end
```

Comment:
yes we talk about that alternative but we like't this better because it look't more readable to use